app/src/main/java/com/soen490chrysalis/papilio/view/UserProfileActivity.kt

```diff
@@ -47,6 +77,16 @@ class UserProfileActivity : AppCompatActivity()
             // Also display the full name and email from the user's Google account on the profile
             binding.userProfileName.setText(acct.displayName)
```

Comment:
Use the firebase object instead to access user attributes & use the property access syntax on Kotlin for getter/setter methods